src/pages/index.html

```diff
@@ -4,16 +4,380 @@
 
 {% block content %}
 
-    <section class="section">
-        <div class="container">
-            <h1>{{ welcome.title }}</h1>
-            <p>{{ welcome.description }}</p>
+<div class="landing ">
+
+  <div class="container  ">
+             <div class="row ">
+              <div class="mb-3 col-xl-7 col-lg-7 col-md-7">
+
+  <div class="modern">
+    <h2>
+      Modern POS that makes running your business easier
+  </h2>
+  <div class="d-flex justify-content-cente align-items-center">
+    
+  <i class="fa-solid fa-check"></i>
+  <p>    Manage cost savings & payments method</p>
+  </div>
+  <div class="d-flex justify-content-cente align-items-center ">
+    <i class="fa-solid fa-check"></i>
+    <p>   Manage cost savings & payments method</p>
+    
+  </div>
+  <div class="trial">
+    <form action="">
+      <input type="email" placeholder="Enter Your Email" />
+      <input type="submit" value="Start Free Trial" />
+    </form>
+    <div class="log">
+<p> Already have an account? <a href="$">Log In now</a> </p>
+
+    </div>
+  </div>
+        
+  </div>
+
+ 
+</div>
+
+  <div class="mb-3 col-xl-5 col-lg-5 col-md-5  position-relative ;
+  ">
+
+  <div class="pic">
+    <div class="shap">
+      <img src="../assets/images/shape (2).png" alt="">
+
+      
```

Comment:
![z](https://github.com/shaban011147/landing/assets/106445705/a4da9bc7-9d84-41cb-8800-13871ee7e7fd)
not responsive in sm breakpoint

Reply:
done